.github/workflows/build.yaml

```diff
@@ -61,8 +61,8 @@ jobs:
     - name: Run Cypress Tests
```

Comment:
Just remove this whole block involving running the Cypress Tests because they aren't working in github actions